### DISCUSSION TITLE
Copilot billing frustration

### DISCUSSION THREAD

**Discussion by lkowalczyk** [community/community#85664] [Copilot Conversations]
### Select Topic Area

General

### Body

When for the second time my card could not be billed for a mysterious reason and my Copilot plan was suspended, I cancelled my subscription to test different solutions.

However, a few days later Github did charge my card but I am still on the free plan. **It has been two weeks**, no one has replied to my tickets.

Anyone else having similar problems with payment processing in Github and not receiving replies from the support? I am now worried I will need to cancel my card.

**Comment by sarahbethd** [comment]
Where did you go to cancel your subscription? For the life of me, I cannot find that so I am going to be issued a new card. I am not happy.

**Comment by lkowalczyk** [comment]
I went to „Settings” in my avatar menu and then „Billing and plans”. There was a cancel button next to my paid plan.

BTW, I even said in the ticket „fine, I give up, I will use  Copilot if you charged me, just enable the plan you made me pay for”. Nothing.

**Comment by lkowalczyk** [comment]
I was finally contacted by the support and the situation was resolved. I still wish, however, that they did not have this many problems with the billing system. I really like this service :)
